src/OpenLoco/Localisation/StringIds.h

```diff
@@ -1813,4 +1813,6 @@ namespace OpenLoco::StringIds
     constexpr string_id tooltip_paint_landscape_tool = 2250;
     constexpr string_id invert_right_mouse_dragging = 2251;
     constexpr string_id tooltip_invert_right_mouse_dragging = 2252;
+    constexpr string_id cash_popup_rendering = 2253;
+    constexpr string_id cash_popup_rendering_tip = 2254;
```

Comment:
the above few tooltips start with `tooltip_*`, so perhaps we should keep this new tooltip consistent with that

Reply:
Good point - I'll change it to `tooltip_cash_popup_randering`